BitwardenShared/UI/Auth/Landing/LandingProcessor.swift

```diff
@@ -22,6 +23,13 @@ class LandingProcessor: StateProcessor<LandingState, LandingAction, LandingEffec
     /// The services required by this processor.
     private let services: Services
 
+    /// Helper class with region specific functions
+    private lazy var regionHelper = RegionHelper(
+        coordinator: coordinator,
+        delegate: self,
+        stateService: services.stateService
+    )
+
```

Comment:
🤔 Why not inject this in the `init` with all the other things? As it's here you need to test what happens inside the `RegionHelper` as well. However, if you inject it you could just mock it.
Additionally, being `lazy` doesn't help much given that is used in `appeared` so basically it's always used almost as soon as the view starts.